### PR TITLE
feat: job postings

### PR DESCRIPTION
**Pull request by livmadsen**
This replaces the blog posts from the starter kit with some pretty rudimentary job postings.
We don't currently need news/blog posts, but I've kept the content type in contentful for reference.

Job posting data is fetched from contentful and rendered with the (very unfinished) field-brand components currently available. Those obvs need some polish still, and so do these pages. Just wanted to add the basics because I had a time gap.

### CODE REVIEW

**Comment by livmadsen** [comment]
Netlify's PR preview is here: https://5c335ef385602d000b7bb547--field-supply-brand.netlify.com/

File: src/pages/team.js [hunk 1]
```diff
@@ -35,6 +90,18 @@ export default TeamIndex
...
+            teaser
+          }
+          slug
+          location
+        }
+      }
+    }
     allContentfulPerson(sort: { fields: [name], order: ASC }) {
       edges {
         node {
```
Comment (gitjul) [inline src/pages/team.js]: ugh, too bad gh makes this red, will make things a bit less pleasant to read

File: src/templates/job-posting.js [hunk 2]
```diff
@@ -3,25 +3,23 @@ import Helmet from 'react-helmet'
...
 
     return (
-      <div style={{ background: '#fff' }}>
-        <Helmet title={`${post.title} | ${siteTitle}`} />
-        <div>
-          <Img alt={post.title} sizes={post.heroImage.sizes} />
-        </div>
+      <div>
+        <Helmet title={`${post.title} | ${siteTitle}`}>
+          <html lang="en" className="theme-fi" />
```
Comment (gitjul) [inline src/templates/job-posting.js]: but in general the convention is singe quotes for everything? (I'm on the OCD side myself 🤓 )

File: src/pages/team.js (hunk 1 above)
Comment (livmadsen) [reply src/pages/team.js]: I wonder why it thinks there's a syntax error there, though. Maybe it's getting tripped up by the text snippet in line 53, could try changing that just to see.

Comment (livmadsen) [reply src/pages/team.js]: Yup 👍 That seems to have been it

**Comment by gitjul** [comment]
how do you generate netflify previews for feat branches?

File: src/templates/job-posting.js (hunk 2 above)
Comment (livmadsen) [reply src/templates/job-posting.js]: Yes, it is. I'm old school enough that it looks bonkers to me to have HTML attributes in single quotes 😅 But that is indeed our convention. I'll update.

**Comment by livmadsen** [comment]
You can configure Netlify to automatically spin up previews for pull requests, or even for every branch in your repo. It's pretty rad.
They actually also have an option to post the preview link as a comment on the PR. I can check if I can easily activate that for here.

**Comment by livmadsen** [comment]
We usually also approve with comments if it's minor things or just suggestions/discussion points. But I might use Request Changes if I notice a bug or if I know of an existing component that can cover some of the task, or even if I worry something is more complex than it needs to be.
In our product repos we also sometimes do it if there are just part of problem or the solution that the reviewer does not fully understand, since that is a strong indication that more comment or context should be added.
For me it's an important distinction that it's not called Reject, but rather Request Changes. Although I agree it comes off as quite aggressive because it's red. I wish GH would change that, honestly.
Along that line, we are also always open for discussions on requests, or for coming up with an alternative or a plan to address the issues later in a follow-up.
In general, we have a fairly well established culture of leaving fulfilling comments on what exactly the change request might be, assuming competence and keeping it phrased as questions or suggestions.

That's sort of our usual Field approach, but I'm totally fine with leaning more heavily to the approve button here and leaving it up to the pull requester (probably you or me anyway 😃) what they want to address or not. These sites are ok to keep pretty low level, we have a tight deadline and I'd even say that a few quick and dirtys are fully allowed within the scope.

**Comment by gitjul** [comment]
Makes sense! And yeah, I used the wrong "Reject" word, probably precisely biased by the color ;) cos I didn't remember the right one.

**Comment by livmadsen** [comment]
With regards to the base.css, that's a leftover from the starter kit. I think we should probably delete the local one here and rather have all shared styles, like resets and body/non-component styles, come from field-brand. Wdyt?

**Pull request merged**